### PR TITLE
"What's New" New brand launch post for 5-17

### PR DESCRIPTION
**Pull request by aalfano7**
This post needs to be live by 8:30AM PT on Tuesday, May 17. 
It is important that it does not go live before 8:00AM PT.
This does not need to be reviewed by a PMM.

**Please keep this file:** [src/content/whats-new/2022/05/whats-new-05-17-new-brand-launch.md](https://github.com/newrelic/docs-website/compare/develop...aalfano7:whats-new-05-17?expand=1#diff-84a7b332671ae9626c6cd55336a62571ca38d2f996836f767def0d3670a17a4e)

**This file is from an old "deleted" branch and is not necessary:** [src/content/whats-new/2022/05/whats-new-05-18-new-brand-launch'.md](https://github.com/newrelic/docs-website/compare/develop...aalfano7:whats-new-05-17?expand=1#diff-55605b92985959f274aa8d8d91cf121be8fff9231048a4d2c3ce26a7b1cb0f06)

### CODE REVIEW

**Review by paperclypse — APPROVED** [review]
Looks good!

**Review by paperclypse — DISMISSED** [review]

**Comment by aalfano7** [comment]
> Update from Lindsay Gordon: we need to update the summary: to remove "Just announced at FutureStack." It should read: 'New Relic reveals its new brand, delivering Data for Engineers to help better plan, build, deploy, and run amazing software.'

> Updated to reflect.

**Review by paperclypse — DISMISSED** [review]
I had to fix a bad image filename @aalfano7. (There was an extra space in the filename) But it should be good now.

File: src/content/whats-new/2022/05/whats-new-05-17-new-brand-launch.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+summary: 'New Relic reveals its new brand, delivering Data for Engineers to help better plan, build, deploy, and run amazing software.' 
+releaseDate: '2022-05-17' 
+learnMoreLink: 'https://newrelic.com/blog/nerd-life/new-brand-announcement'
+---
+
+![Employee sitting at desk, gray and green background](./images/Brand-meta_1200x630.png "Employee sitting at desk, gray background")
+
+## New Relic's bold new brand
+
+Our mission is to help every engineer do their best work every day by using data—not opinions—at every stage of the software lifecycle. At its core, New Relic delivers data for engineers to help you better plan, build, deploy, and run the great software that powers amazing digital experiences for your customers, employees, and partners.
```
Comment (paperclypse) [inline src/content/whats-new/2022/05/whats-new-05-17-new-brand-launch.md:12]:
```suggestion
Our mission is to help every engineer do their best work every day by using data—not opinions—at every stage of the software lifecycle. At our core, New Relic delivers data for engineers to help you better plan, build, deploy, and run the great software that powers amazing digital experiences for your customers, employees, and partners. 
```

**Review by paperclypse — APPROVED** [review]

File: src/content/whats-new/2022/05/whats-new-05-17-new-brand-launch.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: 'Big News—New Relic unveils our bold new brand' 
+summary: 'New Relic reveals its new brand, delivering Data for Engineers to help better plan, build, deploy, and run amazing software.'
```
Comment (paperclypse) [inline src/content/whats-new/2022/05/whats-new-05-17-new-brand-launch.md:3]:
```suggestion
summary: 'New Relic reveals our new brand, delivering Data for Engineers to help better plan, build, deploy, and run amazing software.' 
```

**Review by paperclypse — APPROVED** [review]

**Pull request merged**